infrastructure/main.tf

```diff
@@ -69,3 +79,23 @@ module "api-gateway-web" {
     WEBSITE_NODE_DEFAULT_VERSION = "8.9.4"
   }
 }
+
+// Copy into Azure Key Vault
+
+resource "azurerm_key_vault_secret" "address_lookup_token" {
+  name = "address-lookup-token"
+  value = "${data.vault_generic_secret.address_lookup_token.data["value"]}"
+  vault_uri = "${data.azurerm_key_vault.ccd_shared_key_vault.vault_uri}"
+}
+
+resource "azurerm_key_vault_secret" "oauth2_client_secret" {
+  name = "oauth2-client-secret"
+  value = "${data.vault_generic_secret.oauth2_client_secret.data["value"]}"
+  vault_uri = "${data.azurerm_key_vault.ccd_shared_key_vault.vault_uri}"
+}
+
+resource "azurerm_key_vault_secret" "idam_service_key" {
+  name = "idam-service-key"
```

Comment:
Conflicts with `ccd-admin/idam-service-key` and `ccd-case-print-service/idam-service-key`